runtime/src/main/java/org/corfudb/runtime/clients/AbstractClient.java

```diff
@@ -21,16 +22,21 @@
     @Setter
     private IClientRouter router;
 
+    @Setter
+    private PriorityLevel priorityLevel = PriorityLevel.NORMAL;
```

Comment:
True, but its not needed since it doesn't do any writes and reset doesn't go through the LogUnit.
For the sake of uniformity we should, but before that we need to clean up the client abstractions, they are very convoluted.